tests/test_data/whatsgabycooking.com/whatsgabycooking_1.json

```diff
@@ -0,0 +1,52 @@
+{
+  "author": "Gaby Dalkin",
+  "canonical_url": "https://whatsgabycooking.com/roasted-broccoli/",
+  "site_name": "What's Gaby Cooking",
+  "host": "whatsgabycooking.com",
+  "language": "en-US",
+  "title": "Caesar Parm Roasted Broccoli",
+  "ingredients": [
+    "3-4 heads broccoli (split into florets)",
+    "2 tablespoons olive oil",
+    "Kosher salt and freshly cracked black pepper (to taste)",
+    "1/2 teaspoon red pepper flakes",
+    "2/3 cup freshly grated Parmesan cheese",
+    "2/3 cup panko bread crumbs, sautéed in olive oil",
+    "1 lemon (juiced and zested)",
+    "1/2 cup Tahini Sauce, plus more as needed"
+  ],
+  "instructions": "Preheat the oven to 450 degrees F. Line a baking sheet with parchment paper. Spread the broccoli evenly on the baking sheet and drizzle with 2 tablespoons of olive oil. Using tongs, gently toss the florets in the oil to combine and season with 4 tablespoons of the tahini sauce, salt and pepper and red pepper flakes. Toss to combine.\nTransfer the baking sheet into the oven and roast for 15-20 minutes until the broccoli is just golden and slightly crispy.\nRemove the baking sheet from the oven and drizzle more 2-3 tablespoons more of the tahini sauce on top. Toss with the toasted buttered breadcrumbs, tons of freshly grated parmesan cheese, lemon juice and zest and serve as needed.",
```

Comment:
Should we merge #1211 first so that we can remove these `instructions` keys?

Reply:
We're good to merge this, I'm planning to do a cleanup of all the new stuff and some schema calls once all the open PRs are merged.